src/telliot_core/queries/morphware.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from dataclasses import dataclass
+
+from telliot_core.dtypes.value_type import ValueType
+from telliot_core.queries.abi_query import AbiQuery
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class Morphware(AbiQuery):
+    """Returns the result for a given Morphware query version number.
+
+    Attributes:
+        version:
+            A reference to Morphware data specifications found
+            here: www.TODObyMorphware.org/path/to/data/specs
+
+            More about this query:
+            https://github.com/tellor-io/dataSpecs/blob/main/types/Morphware.md
```

Comment:
Hey, would this be this link instead?
https://github.com/tellor-io/dataSpecs/blob/main/types/AwsSpotPrice.md

Reply:
Good look, I'm going to get rid of `AwsSpotPrice.md` and replace it with `Morphware.md`